tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -384,6 +384,21 @@ module.exports = {
         message = result.value
       })
       return message
+    },
+    /**
+     * clicks the 'copy-public-link-uri' button of a public link
+     *
+     * @param {string} linkName Name of the public link whose URL is to be copied
+     */
+    clickCopyLinkURI: function (linkName) {
```

Comment:
don't use the word `click` on a mobile device we might not "click"
```suggestion
    copyPublicLinkURI: function (linkName) {
```

Reply:
:rocket: